3001-4000/LIT3143Ledata.xml

```diff
@@ -81,13 +90,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <listRelation>
                <relation name="ecrm:P129_is_about" active="LIT3143Ledata" passive="PRS9162TaklaHa"/>
                <relation name="dcterms:isPartOf" active="LIT3143Ledata" passive="LIT1008DossierTH"/>
-                <relation name="lawd:hasAttestation" active="LIT3143Ledata" passive="BNFet138"/>
-                  <relation name="lawd:hasAttestation" active="LIT3143Ledata" passive="IVorlov35"/>
-                    <relation name="lawd:hasAttestation" active="LIT3143Ledata" passive="IVorlov20"/>
-                    <relation name="lawd:hasAttestation" active="LIT3143Ledata" passive="ESath002"/>
-                      <relation name="lawd:hasAttestation" active="LIT3143Ledata" passive="BNFet137"/>
-                        <relation name="lawd:hasAttestation" active="LIT3143Ledata" passive="BNFet50"/>
-                      <relation name="lawd:hasAttestation" active="LIT3143Ledata" passive="BNFet138"/>
+               <relation name="saws:isDifferentTo" active="LIT3143Ledata" passive="LIT6341CCR34"><desc>For the first three 
```

Comment:
I think this relation is wrong, since `LIT3143Ledata `is a general record for any non further specified text of the monastic genealogy of Dabra Libanos

Reply:
OK. Is it useful to mention the relationship to LIT6341CCR34 in some other way here, or is it negligible?